tests/run.sh

```diff
@@ -79,8 +78,28 @@ assert-content "data was ungzipped" "tests/output/input.csv" "1,2,3"
 gunzip tests/output/model.dta.gz
 assert-content "dta file was ungzipped" "tests/output/model.dta" "<stata_dta>"
 
-
 try "load custom libraries" analysis/custom.do
 assert-content "custom command called" "tests/custom.log" "custom command called"
 
+cp tests/fixtures/data.arrow tests/output/data.arrow
+# Load the arrow file; contains:
+# 1000 variables, loaded in batches of 600
+# - date vars: `date` has no nulls, `died_on`` includes nulls
+# - boolean: `flag`
+# - int: `age`, `patient_id` (int64 in feather file, int in stata)
+# - categorical: sex (coded M/F as 0/1, no nulls), region (includes nulls)
+# - num_val: int64 in feather file (values with mean 50,000), converted to long in stata
+try "load arrow file" analysis/arrowload.do
```

Comment:
Ugh, sorry about the custom bash test suite. We previously didn't have a python venv, but now we do, we should probably rewrite this in pytest.

Reply:
I thought about it, but wasn't sure how we'd write pytest tests for something that required the stata sfi module.  Not that I tried it though